scripts/wazuh-alerts-generator/cli.js

```diff
@@ -108,13 +108,7 @@
       'windows',
     ],
     'auditing-policy-monitoring': ['audit', 'ciscat', 'openscap', 'rootcheck'],
-    'thread-detection': [
-      'docker',
-      'mitre',
-      'osquery',
-      'virustotal',
-      'vulnerabilities',
-    ],
+    'thread-detection': ['docker', 'mitre', 'osquery', 'vulnerabilities'],
```

Comment:
**nitpick**: I guess the sample data generator should be able to generate alerts related to VirusTotal.